Sample JSON Rules/change_light_color_and_between_time_of_day.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "If the light is turned on between sunrise and 1 hour after sunrise change the light color",
```

Comment:
For consistency in references, I also recommend you to stick to capability reference syntax, i.e.:

```
"name": "If Switch is On between sunrise and 1 hour after it, Set Color"
```